modules/food-menu/food-item-row.js

```diff
@@ -16,19 +16,22 @@ type FoodItemPropsType = {|
 	style?: any,
 	badgeSpecials?: boolean,
 	spacing: {left: number},
+	onPress: any,
```

Comment:
`onPress: () => any`, please – it needs to be a function, but we don't care what it returns, and we're not passing it anything, so it's function with no args that can return anything.